tests/foreman/conftest.py

```diff
@@ -185,3 +172,32 @@ def pytest_collection_modifyitems(items, config):
 def record_test_timestamp_xml(record_property):
     now = datetime.datetime.utcnow()
     record_property("start_time", now.strftime("%Y-%m-%dT%H:%M:%S"))
+
+
+def pytest_configure(config):
+    """Register custom markers to avoid warnings."""
+    markers = [
+        "stubbed: Tests that are not automates yet.",
+        "deselect(reason=None): Mark test to be removed from collection.",
+        "skip_if_open(issue): Skip test based on issue status.",
+        "tier1: CRUD tests",
+        "tier2: Association tests",
+        "tier3: Systems integration tests",
+        "tier4: Long running tests",
```

Comment:
I know this is not a time and place for this conversation, but I just have to point out that not everyone in the team agrees with these definitions. They aren't aligned with QE org guidelines, either. Personally, I would prefer to leave this whole matter out of here and give tautological explanations, like `tier1: Tier 1 tests`.